### PR TITLE
TIP-426: rename JobInstance parameter `rawConfiguration` to `rawParameters`

### PR DESCRIPTION
**Pull request by momoss**
Rename rawConfiguration to rawParameters:
 - changes the field name in JobInstance ''raw_configuration" to "raw_parameters"
 - keeps the old getRawConfiguration as deprecated 
 - introduces a getRawParameters / setRawParameters
 - add update schema scripts in CE and EE
 - replace all reference of getRawConfiguration by getRawParameters
 - rename JobInstanceType in JobInstanceFormType

With this PR on akeneo-labs/DataGeneratorBundle/pull/39 

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | N
| Changelog updated                 | Y
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) | N
| Migration script                  | Y
| Tech Doc                          | N

### CODE REVIEW

File: CHANGELOG-1.6.md [hunk 1]
```diff
@@ -61,6 +61,9 @@
 
 ## BC breaks
 
+- Move namespace `Pim\Bundle\ImportExportBundle\Form\Type\JobInstanceType` to `Pim\Bundle\ImportExportBundle\Form\Type\JobInstanceFormType`
```
Comment (skeleton) [inline CHANGELOG-1.6.md]: - Rename `Pim\Bundle\ImportExportBundle\Form\Type\JobInstanceType` to `Pim\Bundle\ImportExportBundle\Form\Type\JobInstanceFormType`

Comment (skeleton) [inline CHANGELOG-1.6.md]: add a sed command in UPGRADE-1.6 file

File: src/Akeneo/Bundle/BatchBundle/Resources/config/model/doctrine/JobInstance.orm.yml [hunk 2]
```diff
@@ -35,9 +35,9 @@ Akeneo\Component\Batch\Model\JobInstance:
             type: string
             length: 255
             column: connector
-        rawConfiguration:
+        rawParameters:
```
Comment (skeleton) [inline src/Akeneo/Bundle/BatchBundle/Resources/config/model/doctrine/JobInstance.orm.yml]: as you change the model, you have to add a migration script

File: src/Akeneo/Component/Batch/Model/JobInstance.php [hunk 3]
```diff
@@ -59,7 +59,7 @@ class JobInstance
     protected $type;
 
     /** @var array */
-    protected $rawConfiguration = array();
+    protected $rawParameters = array();
```
Comment (skeleton) [inline src/Akeneo/Component/Batch/Model/JobInstance.php]: []

File: src/Akeneo/Component/Batch/Model/JobInstance.php [hunk 4]
```diff
@@ -218,29 +218,55 @@ public function getType()
     }
 
     /**
-     * This configuration can be used to create a JobParameters, stored like this in a legacy way
-     * TODO TIP-426: we should rename this configuration to rawParameters
+     * This parameters can be used to create a JobParameters, stored like this in a legacy way
      *
-     * @param array $configuration
+     * @param array $parameters
```
Comment (skeleton) [inline src/Akeneo/Component/Batch/Model/JobInstance.php]: $rowParameters

File: src/Pim/Bundle/ImportExportBundle/Controller/JobProfileController.php [hunk 5]
```diff
@@ -134,8 +134,8 @@ public function __construct(
         $this->jobRegistry           = $jobRegistry;
         $this->jobType               = $jobType;
 
-        $this->jobInstanceType       = $jobInstanceType;
-        $this->jobInstanceType->setJobType($this->jobType);
+        $this->jobInstanceFormType       = $jobInstanceFormType;
```
Comment (skeleton) [inline src/Pim/Bundle/ImportExportBundle/Controller/JobProfileController.php]: #YoloAlignment :p

**Comment by skeleton** [comment]
add migration script in EE please
GTM once fixed

**Comment by pierallard** [comment]
GTM

**Pull request merged**